src/hammer-vlsi/hammer_vlsi/driver.py

```diff
@@ -121,24 +121,30 @@ def update_project_configs(self, project_configs: List[dict]) -> None:
         self.database.update_project(self.project_configs)
 
     def load_technology(self, cache_dir: str = "") -> None:
-        tech_str = self.database.get_setting("vlsi.core.technology")
+        tech_str = self.database.get_setting("vlsi.core.technology") # type: str
 
         if cache_dir == "":
             cache_dir = os.path.join(self.obj_dir, "tech-%s-cache" % tech_str)
 
         tech_paths = self.database.get_setting("vlsi.core.technology_path")
-        tech_json_path = ""  # type: str
+        tech_blob_path = ""  # type: str
+        is_yaml = False # type: bool
         for path in tech_paths:
-            tech_json_path = os.path.join(path, tech_str, "%s.tech.json" % tech_str)
-            if os.path.exists(tech_json_path):
+            tech_blob_path = os.path.join(path, tech_str, "%s.tech.json" % tech_str)
+            if os.path.exists(tech_blob_path):
+                is_yaml = False
                 break
-        if tech_json_path == "":
-            self.log.error("Technology {0} not found or missing .tech.json!".format(tech_str))
+            tech_blob_path = os.path.join(path, tech_str, "%s.tech.yaml" % tech_str)
+            if os.path.exists(tech_blob_path):
+                is_yaml = True
```

Comment:
Generally looks good - I would move the `is_yaml` detection logic inside `load_from_dir` though. (e.g. when we load `defaults.yml/json` the exact one is determined inside the load plugin logic)